packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -181,6 +190,31 @@ abstract contract MajorityVotingBase is
         return _canExecute(_proposalId);
     }
 
+    /// @inheritdoc IMajorityVoting
+    function support(uint256 _proposalId) public view virtual returns (uint256) {
+        Proposal storage proposal_ = proposals[_proposalId];
+
+        return _calculatePct(proposal_.yes, proposal_.yes + proposal_.no);
```

Comment:
Careful about percentages. The SDK is prepared to encode `0-1` ratios with an arbitrary precision (digits).

We may not enlarge the scope of this PR, but I would suggest making this compatible on day 0 if possible. i.e. the plugin has a constant/global number of 5 digits to express the support ratio, hence values could range between `0 ... 55000 ... 100000`

Reply:
This will happen in https://aragonassociation.atlassian.net/browse/APP-1264